features/eolearn/features/doubly_logistic_approximation.py

```diff
@@ -9,53 +9,50 @@
 file in the root directory of this source tree.
 """
 import itertools as it
+from typing import List, Optional
 
 import numpy as np
 from scipy.optimize import curve_fit
 
-from eolearn.core import EOTask, FeatureType
+from eolearn.core import EOPatch, EOTask, FeatureType
+from eolearn.core.utils.parsing import SingleFeatureSpec
 
 
-def doubly_logistic(middle, initial_value, scale, a1, a2, a3, a4, a5):
-    # pylint: disable=invalid-name,locally-disabled
-    """
-    Function that is passed to `scipy.optimize`
-    """
-    return initial_value + scale * np.piecewise(
-        middle,
-        [middle < a1, middle >= a1],
-        [lambda y: np.exp(-(((a1 - y) / a4) ** a5)), lambda y: np.exp(-(((y - a1) / a2) ** a3))],
-    )
+def doubly_logistic(middle, initial_value, scale, a1, a2, a3, a4, a5) -> np.ndarray:
```

Comment:
```suggestion
def doubly_logistic(
    middle: float, initial_value: float, scale: float, a1: float, a2: float, a3: float, a4: float, a5: float
) -> np.ndarray:
```

Reply:
I planned to do thins in the original. But I am very sure some of these are meant to be numpy.ndarrays. It is not clear which ones or if any